mmv1/third_party/terraform/.teamcity/components/generated/project.erb

```diff
@@ -59,8 +59,12 @@ fun Google<%= version.capitalize unless version == 'ga' -%>(environment: String,
         // Adding this allows custom builds to use alternative branches. E.g. testing release branches in the downstreams
 
         params {
+            // Controls the VCS root, and allows custom builds to point at different branches
             param("BRANCH_NAME", branchRef)
-            
+<% unless version == 'ga' -%>
+            // Skip the sweeper for project resources in the Beta tests only
+            param("env.SKIP_PROJECT_SWEEPER", 1)
```

Comment:
I'm not sure if this is possible using .teamcity files, but we may want to also make sure that both Post-sweepers use  `SKIP_PROJECT_SWEEPER`. I think the GA tests will usually finish while the Beta tests are still running, so without SKIP_PROJECT_SWEEPER, we could end up with sporadic test failures in the Beta runs.